android/app/src/main/AndroidManifest.xml

```diff
@@ -6,6 +6,7 @@
         android:maxSdkVersion="28" />
     <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
     <uses-permission android:name="android.permission.INTERNET" />
+    <uses-permission android:name="android.permission.QUERY_ALL_PACKAGES" />
```

Comment:
Please do not use this permission.
Here is a guide: https://developer.android.com/training/package-visibility
https://developer.android.com/training/package-visibility/use-cases